content/docs/ops/bosh-releases-in-15-minutes-with-ubuntu.md

```diff
@@ -0,0 +1,143 @@
+Configure the BOSH command line client to connect to the new director running in Vagrant.  The default username is `admin` and password is `admin`:
+```shell
+bosh target 192.168.50.4 lite
+```
+
+### Add Routes
+The VirtualMachine is up and will create containers to service our BOSH requests.  In order for these containers to communicate with the network, a route needs to be added:
+
+```shell
+sudo ./bin/add-route
+```
+
+### Inspect the Director
+The director is up and running and can be inspected to review the configuration.  Currently the director is empty since nothing has been configured or uploaded:
+```shell
+bosh releases
+bosh stemcells
+bosh deployments
+```
+
+### Upload a Release
+[cg-secureproxy-boshrelease](https://github.com/18F/cg-secureproxy-boshrelease) is a BOSH release the cloud.gov team maintains and is a good example.  Clone the release, create a BOSH release and upload it to the director:
+```shell
+cd $HOME/workspace
+git clone https://github.com/18F/cg-secureproxy-boshrelease.git
+bosh create release
+bosh upload release
+```
+
+### Upload a Stemcell
+There is now a `secureproxy` BOSH release, but a operating system is needed to run it on.  These images are called `stemcells` and can be found at [bosh.io](https://bosh.io/stemcells):
+```shell
+cd $HOME/workspace
+curl -O https://s3.amazonaws.com/bosh-core-stemcells/warden/bosh-stemcell-3312.12-warden-boshlite-ubuntu-trusty-go_agent.tgz
+bosh upload stemcell ./bosh-stemcell-3312.12-warden-boshlite-ubuntu-trusty-go_agent.tgz
+```
+
+### Create a Deployment
+There is now a `secureproxy` BOSH release and a `Ubuntu Trusty` stemcell.  A deployment manifest tells the director which combination of releases, jobs and stemcells need to be deployed.  Each release usually contains a sample deployment manifest or tool to generate one:
+```shell
+cd $HOME/workspace/cg-secureproxy-boshrelease/templates
+./make_manifest warden
+```
+
+### Set Required Parameters
+Look at the manifest (by default located at `$HOME/workspace/cg-secureproxy-boshrelease/tmp/secureproxy-warden-manifest.yml`) and find the line which starts with `properties:` and add a secureproxy section with `listen_port` and `proxy_port`.  Since this is an example these won't be actually used, so lets just use 9292 and 9293 for now:
+```
+properties:
+  secureproxy:
+      listen_port: 9292
+          proxy_port: 9293
+```
+
+To see all of the variables a release might use, locate the spec file which describes required, optional, and defaults (ex `$HOME/workspace/cg-secureproxy-boshrelease/jobs/secureproxy/spec`)
+
+### Deploy the Manifest
+There is now a BOSH secureproxy release, Ubuntu stemcell, and secureproxy deployment manifest.  To deploy the manifest:
+```shell
+bosh deploy
+```
+
+### The Job is Running
+The secureproxy job is now running and proxying requests between `listen_port` and `proxy_port`.  To inspect the state of the director:
+```shell
+bosh releases
+bosh stemcells
+bosh deployments
+bosh vms
+```
```

Comment:
Any thing to pull from the output?
How would these commands help with debugging?